coffea/processor/executor.py

```diff
@@ -1457,11 +1627,21 @@ def run_uproot_job(
     else:
         closure = partial(closure, processor_instance=pi_to_send)
 
+    if dynamic_chunksize:
+        events_total = sum(f.metadata["numentries"] for f in fileset)
+    else:
+        chunks = [c for c in chunks]
+        events_total = sum(c.entrystop - c.entrystart for c in chunks)
+
     exe_args = {
-        "unit": "chunk",
+        "unit": "event" if executor is work_queue_executor else "chunk",
         "function_name": type(processor_instance).__name__,
         "use_dataframes": use_dataframes,
+        "events-total": events_total,
```

Comment:
I think all the other arguments use underscore, would be good for these to as well for consistency.